spec/cqm/converter/hds_record_spec.rb

```diff
@@ -39,49 +38,49 @@
 
   it 'converts HDS EH 1 to QDM EH 1 properly' do
```

Comment:
Looks like several of these test names may need to be adjusted depending on what is being tested.